### ISSUE TITLE
Make docker creds section in network.yaml as optional in Hyperledger Indy

### ISSUE THREAD

**Issue by jagpreetsinghsasan** [hyperledger-labs/blockchain-automation-framework#616]
Currently, we don't have an option to exclude docker credentials from network.yaml
The docker credentials are used to create a kubernetes secret (docker cred template) which is used to pull images with docker credentials.
With the inclusion of images over a public dockerhub registry, we dont need the docker username & password section in the network.yaml, the regcred and thus there is no need to pull the image with docker creds.

The changes required are:

- In the file **hyperledger-indy/configuration/roles/create/imagepullsecret/tasks/main.yaml**, change the first task named "**Check for ImagePullSecret for {{ organization }}**" and the next task named **"Create the docker pull registry secret for {{ component_ns }}**" to run only if network.docker.username && network.docker.password are defined and are not empty. To check if a variable (network.docker.username) is defined and not empty, you can use the code `network.docker.username | default('', true) | trim != ''`

- For the charts, you need to a put a condition to use **imagePullSecrets** only when `network.docker.username | default('', true) | trim != ''` and `network.docker.password | default('', true) | trim != ''`

	The following charts are required to be changed for Hyperledger Indy

	- [ ] indy-auth-job chart at line 13

	- [ ] indy-key-mgmt chart at line 13

	- [ ] indy-ledger-txn chart at line 21

	- [ ] indy-node chart at line 149

- Test the network with the above mentioned changes

**Comment by Roshan13046** [comment]
I would like to fix this issue. Can I raise a PR for this?

**Comment by sownak** [comment]
@Roshan13046 yes, of course you can. Please follow our contributing guidelines.
